reverse.sh

```diff
@@ -0,0 +1,12 @@
+count=0
+array=()
```

Comment:
It seems I needed to add 
!#/usr/bin/env bash
to make sure it was run using bash instead of sh